### DISCUSSION TITLE
Receiver graph

### DISCUSSION THREAD

**Discussion by Phoenixoss** [vradarserver/vrs#92] [Q&A]
Hi, 
I am new in area of VRS.
I have installed it on my RPi4 with .sh script.
After restart, I realized that no aircraft pictures nor aircraft icons on map (I have silhouettes)
Second, I cannot create receiver graph due to lack option for that in MENU :(

![vrs_error](https://user-images.githubusercontent.com/106580679/219056527-ec75b7b4-16f5-4427-b507-bdadca562934.JPG)
![mono_error](https://user-images.githubusercontent.com/106580679/219057089-c84763ef-2077-409c-82c4-fe13527103f4.JPG)

What should I do? :(
Regards, Laszlo

PS: 
VRS: 2.4.4.36598 (20/12/2019 20:19:56)
Mono: 64 bit

**Comment by BallerG** [comment]
Aircraft pictures are not working due to an outage at airport-data which is where they come from. The site has been down most of this week, and it's unknown when it'll be back.

**Reply by vradarserver** [reply]
Further to that, the current previews of VRS won't fetch aircraft pictures from airport-data any more due to some concerns over airport-data.com (https://github.com/vradarserver/vrs/issues/38), but there is a plugin in the custom plugins repository that will restore them.

> no [...] aircraft icons on map

If you've installed v2 then I would maybe try the v3 preview. In version 2 the server is responsible for drawing the aircraft markers on behalf of the browser. If Mono is having any kind of issue with graphics handling then you can end up with either the server crashing or images not getting drawn.

In version 3 the markers are drawn by the client's browser as SVG vector graphics. You can switch back to server-drawn graphics if you want to, but in your case it means that if Mono has an issue with rendering images then it doesn't affect aircraft markers because Mono isn't drawing those any more.

> I cannot create receiver graph due to lack option for that in MENU

You need to specify a receiver location. The range graph will be centred on that location.

**Reply by Phoenixoss** [reply]
Thank you, it helped!
